packages/chart/src/ChartModel.ts

```diff
@@ -28,11 +29,14 @@ class ChartModel {
 
   static EVENT_LOADFINISHED = 'ChartModel.EVENT_LOADFINISHED';
 
-  constructor() {
+  constructor(dh: DhType) {
+    this.dh = dh;
```

Comment:
I'm not big on this, but I think this reveals a separate issue... our `ChartModel` right now isn't abstract, as it requires the Deephaven API (and also outputs as Plotly, which could be a further abstraction but is alright for now...).
The issue is that `FigureChartModel` needs the formatter settings, and `Formatter` itself depends on our DH API. In a perfect world, `Formatter` interface wouldn't be dependent on the DH API (as that's an implementation detail), and the chart model itself would manage it's own formatting (and listening for formatter updates) rather than `Chart.tsx` handling the settings change and setting it on the `ChartModel`...

That being said, now's not the time to change that abstraction, model was already tied to the JS API because of the formatter settings. So leave it like this, but something to consider for the future...

Reply:
Agreed, makes sense.